### PR TITLE
Update vertx dependencies

### PR DESCRIPTION
**Pull request by OwenCorrigan76**
Update Vertx dependencies from version 3.7.1 to version 4.3.7.

### CODE REVIEW

File: Makefile
Comment (kyguy) [inline Makefile]: extra line

File: java/http/deployment-tracing-opentelemetry.yaml
Comment (kyguy) [inline java/http/deployment-tracing-opentelemetry.yaml]: Stale

File: java/http/deployment-tracing-opentelemetry.yaml
Comment (kyguy) [inline java/http/deployment-tracing-opentelemetry.yaml]: Stale

File: java/http/java-http-consumer.yaml
Comment (kyguy) [inline java/http/java-http-consumer.yaml]: Stale

File: java/http/java-http-producer.yaml
Comment (kyguy) [inline java/http/java-http-producer.yaml]: Stale

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: My cause checkstyle errors, avoid wildcard imports and import each lib individual

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:53]: extra line

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:72]: extra line

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: extra spaces

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: undo indentation changes

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Fix indentation

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: fix indentation

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: fix indentation

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:254]: extra line

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:257]: extra line

File: java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java]: extra line

File: Makefile
Comment (OwenCorrigan76) [reply Makefile]: Removed

File: java/http/deployment-tracing-opentelemetry.yaml
Comment (OwenCorrigan76) [reply java/http/deployment-tracing-opentelemetry.yaml]: This was removed on the last PR. Not sure how it got in again. Now removed.

File: java/http/deployment-tracing-opentelemetry.yaml
Comment (OwenCorrigan76) [reply java/http/deployment-tracing-opentelemetry.yaml]: Same as above - now removed.

File: java/http/java-http-consumer.yaml
Comment (OwenCorrigan76) [reply java/http/java-http-consumer.yaml]: Same as above - now removed

File: java/http/java-http-producer.yaml
Comment (OwenCorrigan76) [reply java/http/java-http-producer.yaml]: Same as above - now removed

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Fixed

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Fixed??

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:254]: Indentation on an empty line??

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:257]: The empty line was on 254, now on 255

File: java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java]: Removed

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Fixed

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Fixed (I think)

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Fixed

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:72]: ?? There's a line space in the original??

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:53]: ?? Same as original

File: java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java
Comment (ppatierno) [reply java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java]: We avoid to have `*` when importing classes but listing them all.

**Review by ppatierno — CHANGES_REQUESTED** [review]

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Removed wildcard and used individual specific imports

File: java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-producer/src/main/java/HttpKafkaProducer.java]: All wildcard imports removed and replaced with appropriate imports

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (scholzj) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: This seems to be misaligned -> the indentation of line 174 and 180 is different from 175-179.

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (scholzj) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Just a tip: if you leave this method where the original was, the diff will look nicer and reviews will be much easier for reviewers as they would see the old and new code next to each other. There does not seem to be any real reason why to move this method within the same class.

**Review by scholzj — COMMENTED** [review]
Left some nits. But looks mostly good.

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Not sure why I moved the subscribe method. Put back to correct place.

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: I've realigned the whole HttpKafkaConsumer class to where it was in the previous commits. Hopefully this is correct?

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]
LGTM

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Extra indentation

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (kyguy) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java:53]: This is fine, just deleted an extra space, ignore this comment

**Review by kyguy — APPROVED** [review]
Looks great @OwenCorrigan76, well done!

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java
Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: Fixed

**Comment by scholzj** [comment]
@OwenCorrigan76 We already approved it -> why does it need another review?

**Review by kyguy — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]
Approving again ;-)

**Comment by OwenCorrigan76** [comment]
@ppatierno @scholzj  Sorry re-requested review after you both approved. My bad!

**Review by scholzj — APPROVED** [review]

**Pull request merged**
